### DISCUSSION TITLE
A Windows BLE tool using this plugin?

### DISCUSSION THREAD

**Discussion by AskBojesen** [dotnet-bluetooth-le/dotnet-bluetooth-le#768] [General]
Hi BLE Developers!

The only BLE tool for Windows - which I could find exploring BLE in a general way is this:
https://github.com/microsoft/BluetoothLEExplorer

I have another Windows application used by myself in our BLE development.
The Windows application use the interface from this BLE plugin.
This application could go open-source if anybody are interested in such ?

(Besides me...)

![image](https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/assets/131768758/2867e286-e585-4ab5-bbf4-c2bd112ef8b1)

**Comment by janusw** [comment]
> I have another Windows application used by myself in our BLE development. The Windows application use the interface from this BLE plugin. This application could go open-source if anybody are interested in such ?

Well, it this was cross-platform it would definitely be interesting for the dotnet-bluetooth-le project as a sample/tool. But since it is Windows-only, I wouldn't want to include it in the dotnet-bluetooth-le repo directly.

Of course you can simply publish it on your personal GitHub account. Alternatively I could offer to include it in the dotnet-bluetooth-le organization as a separate repo and give you maintainer permissions. Might improve the outreach and visibility.

I cannot tell you if there is much community interest in such a tool. I personally don't have too much interest in the Windows platform (my focus is more on the mobile platforms right now, i.e. Android and iOS). But of course the .NET community at large is, at least historically, quite Windows-centric, so I can image that a good number of dotnet-bluetooth-le users might be interested in such a tool after all ...

**Comment by AskBojesen** [comment]
If nobody are interested in this - then I will keep it for myself :-)
